src/D2L.Bmx/ConsoleWriter.cs

```diff
@@ -0,0 +1,17 @@
+namespace D2L.Bmx;
+
+internal interface IConsoleWriter {
```

Comment:
We're reasonably close to a point where some unit tests around our prompting and printing logic would be worthwhile.
I could really use a few when I was writing up https://github.com/Brightspace/bmx/pull/463
Interfaces are handy then.